tests/unit/conditions/test_return_value.py

```diff
@@ -142,6 +143,23 @@ def test_return_value_test_with_context_variable_cant_run_eval():
         test.eval(0)
 
 
+def test_return_value_test_with_resolved_context():
+    test = ReturnValueTest(comparator="==", value=":foo")
+    context = {":foo": 1234}
+
+    resolved = test.with_resolved_context(**context)
+    assert resolved.comparator == test.comparator
+    assert resolved.index == test.index
+    assert resolved.value == _resolve_context_variable(test.value, **context)
+
+    test = ReturnValueTest(comparator="==", value=[42, ":foo"])
+
+    resolved = test.with_resolved_context(**context)
+    assert resolved.comparator == test.comparator
+    assert resolved.index == test.index
+    assert resolved.value == _resolve_context_variable(test.value, **context)
```

Comment:
Same comment as above...?
```suggestion
    assert resolved.value == [42, context[":foo"]]
```